modules/client/examples/01-client/12-prepare-uninstallation.ts

```diff
@@ -24,17 +20,17 @@ const client: Client = new Client(context);
 
 const prepareUninstallationParams: PrepareUninstallationParams = {
   daoAddressOrEns: "0x1234567890123456789012345678901234567890", // my-dao.dao.eth
-  versionTag: {
-    build: 1,
-    release: 1,
-  },
   pluginAddress: "0x2345678901234567890123456789012345678901",
+  uninstallationParams: [[
+    "casa",
+  ]],
```

Comment:
Why an array of arrays?

Reply:
Just an example this must be anything that matches the abi on the plugin metadata, from a single value, to an array of values or object